interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/out/ServiceRequestOutboundInterceptorTest.java

```diff
@@ -73,24 +49,24 @@ void testHandleMessage() throws IOException {
         SdtContext.getContext().setServiceRequestId(1L);
         String data = "<xml>content</xml>";
         final SoapMessage soapMessage = getDummySoapMessageWithCachedOutputStream(data);
-        final ServiceRequestOutboundInterceptor serviceRequestOutboundInterceptor = new ServiceRequestOutboundInterceptor(mockServiceRequestDao);
+        final ServiceRequestOutboundInterceptor serviceRequestOutboundInterceptor = new ServiceRequestOutboundInterceptor(requestDaoService);
         final ServiceRequest serviceRequest = new ServiceRequest();
         when(mockServiceRequestDao.fetch(ServiceRequest.class, 1L)).thenReturn(serviceRequest);
-        mockServiceRequestDao.persist(serviceRequest);
 
-        Assertions.assertNull(serviceRequest.getResponseDateTime());
-        Assertions.assertNull(serviceRequest.getResponsePayload());
+        assertNull(serviceRequest.getResponseDateTime());
+        assertNull(serviceRequest.getResponsePayload());
         serviceRequestOutboundInterceptor.handleMessage(soapMessage);
-        Assertions.assertNotNull(serviceRequest.getResponseDateTime());
-        Assertions.assertTrue(serviceRequest.getResponsePayload().contains(data));
+        assertNotNull(serviceRequest.getResponseDateTime());
+        String response = new String(serviceRequest.getResponsePayload(), StandardCharsets.UTF_8);
+        assertTrue(response.contains(data));
```

Comment:
Should be able to add `verify(mockServiceRequestDao).persist(serviceRequest);` here to check that persist() was actually called.